Edgetracking algorithm (#61)
JuliaDynamics/Attractors.jl: docs/src/index.md

## Latest news

> - New functions [`edgetracking`](@ref) and [`bisect_to_edge`](@ref) added that implement the **edge tracking algorithm**, a method to find saddles, *edge states* or *Melancholia states* in dynamical systems, also when they are unstable chaotic sets.

Reviewer: ```suggestion
- New functions [`edgetracking`](@ref) and [`bisect_to_edge`](@ref) added that implement an **edge tracking algorithm**, a method to find saddles or *edge states* in dynamical systems, also when they are unstable chaotic sets.
```

there are many algorithms, such as the saddle stradle, and this is one of them. (not "the")

(Also, let's stick to commonly accepted terminology and keep the docstring lean, it is fine to refer to this term in the rest of the docs)
